flink-tests/src/test/java/org/apache/flink/test/misc/AttemptNumberITCase.java

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
In order to reduce the number of redundant (but time consuming) tests: Can you fuse this test with the TaskFailureITCase?